build/mark-stable-release.sh

```diff
@@ -28,7 +32,7 @@ fi
 
 STABLE_FILE_LOCATION="kubernetes-release/release/stable.txt"
 
-version_file=$(mktemp -t stable)
+version_file=$(mktemp -t stable.XXXXXX)
```

Comment:
This gives a funny dir name on OS X: stable.XXXXXX.bRtDPfQv but it works so that's probably fine.